addons/web_editor/static/src/js/editor/odoo-editor/src/utils/sanitize.js

```diff
@@ -18,56 +18,50 @@ import {
     getUrlsInfosInString,
     URL_REGEX,
     isVoidElement,
+    startPos,
+    ancestors,
 } from './utils.js';
 
 const NOT_A_NUMBER = /[^\d]/g;
+
+function _hasPseudoElementContent (node, pseudoSelector) {
```

Comment:
This underscore can be removed. It serves no purpose since the function is not exported.